hydra-plutus/src/Hydra/Contract/Initial.hs

```diff
@@ -106,7 +106,9 @@ checkCommit commitValidator headId committedRefs context =
  where
   checkCommittedValue =
     traceIfFalse $(errorCode LockedValueDoesNotMatch) $
-      lockedValue == initialValue + committedValue
+      -- NOTE: Ada in initialValue is usually lower than in the locked ADA due
+      -- to higher deposit needed for commit output than for initial output
+      lockedValue `geq` (initialValue + committedValue)
```

Comment:
I did not see a change to the `Commit` mutations exposing this, am I missing something?

Reply:
The healthy case will fail with == now.